examples/Vendored Framework Example/Vendored Framework Example.xcodeproj/project.pbxproj

```diff
@@ -335,6 +335,7 @@
 			baseConfigurationReference = A8A706EB2A5031760E4C935B /* Pods-Vendored Framework Example.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				EXCLUDED_ARCHS = arm64;
```

Comment:
For this example we consume a vendored framework that does not ship this architecture. Excluding it is fine.